### PR TITLE
docs(templates): add nullish attributes

### PR DESCRIPTION
**Pull request by Garbee**
# Description

Add a section for attributes detailing how nullish values will work.
This is useful in very specific situations when working with templates.

## Motivation & context

I was attempting to set `aria-hidden` on an element when it wasn't viewable within a component.
However, I didn't find a way documented to fulfill what is needed in this situation.
Upon asking in the support chat I was provided a solution by using nullish values.

Seeing as that resolved my issue, I figured I'd document it for anyone else hitting this.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.
- [x] **Docs**: A change that modifies documentation

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by Garbee** [comment]
When I ran this locally, I did have an error which seemed to emanate from https://github.com/microsoft/fast/blob/c86b63bd5323880f0dd859cbd2bf8a20ca1ccba0/packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts#L7 . I couldn't get it to go away by editing the `<prefix>` tag in the comment. But editing the generated file manually during the server run got it to let me see the changes.

File: packages/web-components/fast-element/docs/guide/declaring-templates.md [hunk 1]
```diff
@@ -202,4 +210,4 @@ In the future we're planning new optimizations that will enable us to safely det
 
 In most cases, the template that `FASTElement` renders is determined by the `template` property of the Custom Element's configuration. However, you can also implement a method on your Custom Element class named `resolveTemplate()` that returns a template instance. If this method is present, it will be called during `connectedCallback` to obtain the template to use. This allows the element author to dynamically select completely different templates based on the state of the element at the time of connection.
 
-In addition to dynamic template selection during the `connectedCallback`, the `$fastController` property of `FASTElement` enables dynamically changing the template at any time though setting the controller's `template` property to any valid template.
\ No newline at end of file
+In addition to dynamic template selection during the `connectedCallback`, the `$fastController` property of `FASTElement` enables dynamically changing the template at any time though setting the controller's `template` property to any valid template.
```
Comment (EisenbergEffect) [inline packages/web-components/fast-element/docs/guide/declaring-templates.md:213]:
```suggestion
In addition to dynamic template selection during the `connectedCallback`, the `$fastController` property of `FASTElement` enables dynamically changing the template at any time through setting the controller's `template` property to any valid template.
```

Comment (EisenbergEffect) [reply packages/web-components/fast-element/docs/guide/declaring-templates.md:213]: Just happened to catch this other type while reviewing.

**Review by EisenbergEffect — APPROVED** [review]
Nice improvement. Thanks!

**Comment by Garbee** [comment]
Applied the typo fix. Good eye.

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**